Library/Formula/mapext.rb

```diff
@@ -0,0 +1,18 @@
+class Mapext < Formula
+  desc "map files from one extension to another via a specified command"
+  homepage "http://www.acme.com/software/mapext/"
+  url "http://www.acme.com/software/mapext/mapext_10nov1995.tar.gz"
```

Comment:
Well, true. In the course of making my first formulas I thought why not. Feel free to close this PR, I'm not particularly keen on seeing it merged. Thought someone/the archive of formulas itself might benefit but plase just go ahead and close, fine with me!